docs/guide/tools/geometry.md

```diff
@@ -0,0 +1,78 @@
+# Geometry tools
+
+In the latest development version, there is a setsquare tool. A compass wheel is in work as well.
+
+## Setsquare
+
+The setsquare can be used for measuring lengths and angles and for geometric constructions.
+
+![Setsquare-Overview](../../img/guide/setsquare_overview.png)
+
+The longest side of the setsquare is used for measuring lengths and to draw lines aligned to it.
+Units are centimeters for lengths and degrees for angles. There are marks (in green) parallel 
+to the longest side of the setsquare, that allow measuring distances from that side.
+
+Angular marks (in purple) help measuring angles. There is also a display of the current angle
+(in tenth of degrees) between the longest side of the setsquare and a horizontal line.
+The number always looks up. On the picture this angle is 10.0 degrees.
+
+The shorter two sides of the setsquare are not used for drawing lines aligned to them. Instead they
+are used for drawing lines to the middle point of the longest side of the setsquare.
+
+### Setup and activation
+
+In order to setup the setsquare, check your Display DPI Calibration in the Zoom panel of menu
+`Edit > Preferences` to ensure that the length on your screen matches the physical length. Set
+the zoom to 100% to profit from the calibration.
+
+Switch the setsquare on/off either from menu `Edit > Setsquare` or via the toolbar (setsquare symbol).
+The setsquare is always attached to one particular page of the document. If you need the setsquare on
+a different page of the document, deactivate it on the old one and activate it on the new one. This can
+be achieved by clicking twice on the setsquare symbol on the toolbar.
+
+The setsquare is designed to be used either with touch (one hand) and stylus/mouse (other hand) or with
+keyboard (one hand) and stylus/mouse (other hand).
+
+**Keyboard and Touch** are used to move the setsquare (and the page) around, to rotate it and change
+its size, whereas **Stylus/Mouse** are used to draw/annotate and in particular to draw lines at the
+longest side of the setsquare.
+
+### Keyboard control
+
+Several keys can be used to move, rotate and scale the setsquare. The `Alt` modifier can be used to get smaller steps.
+
+- `arrow keys` for moving around. Steps are 0.25 cm (big) and 0.25 mm (small). Use the shift key to make the setsquare move parallel or perpendicular to the longest side of the setsquare.
+- `r/R` for rotation (countclockwise/clockwise) around the midpoint of the longest side of the setsquare. Steps are 5 degrees (big) and 0.2 degrees (small)
+- `s/S` for scaling (up/down) from the midpoint of the longest side of the setsquare. Steps for scaling up are 10% (big) and 1% (small). Scaling down has the opposite effect.
+
+### Touch control
+
+Touch can be used with one finger or two fingers. Gestures with more fingers are not handled.
```

Comment:
Does this depend on the Touch Drawing setting?

Reply:
When Touch Drawing is enabled, you still draw outside of the geometry tool. Otherwise there is no difference. I have updated the description.